From 54cc16821b952f42c14f9ab03fcef19397dbac2d Mon Sep 17 00:00:00 2001
From: David Daney <david.daney@cavium.com>
Date: Tue, 20 May 2014 11:29:21 -0700
Subject: [PATCH 090/202] MIPS: OCTEON: Disable cn78XX PKI_GEN_INT[DRP_NOAVAIL]

It is causing too much noise.

Signed-off-by: David Daney <david.daney@cavium.com>
[Original patch taken from Cavium SDK 3.1.1-544]
Signed-off-by: Bin Jiang <bin.jiang@windriver.com>
---
 arch/mips/cavium-octeon/octeon-78xx-errors.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/mips/cavium-octeon/octeon-78xx-errors.c b/arch/mips/cavium-octeon/octeon-78xx-errors.c
index dfdf2c9..6ca5d20 100644
--- a/arch/mips/cavium-octeon/octeon-78xx-errors.c
+++ b/arch/mips/cavium-octeon/octeon-78xx-errors.c
@@ -1821,7 +1821,6 @@ struct cvmx_error_78xx error_array_cn78xxp1[] = {
 	{ 0x1180044000020ull,	"Error: CVMX_PKI_GEN_INT[%llx][%u]",	0x44003,	0,	CVMX_ERROR_GROUP_INTERNAL,	3}	/* SOP */,
 	{ 0x1180044000020ull,	"Error: CVMX_PKI_GEN_INT[%llx][%u]",	0x44004,	0,	CVMX_ERROR_GROUP_INTERNAL,	4}	/* EOP */,
 	{ 0x1180044000020ull,	"Error: CVMX_PKI_GEN_INT[%llx][%u]",	0x44005,	0,	CVMX_ERROR_GROUP_INTERNAL,	5}	/* DAT */,
-	{ 0x1180044000020ull,	"Error: CVMX_PKI_GEN_INT[%llx][%u]",	0x44006,	0,	CVMX_ERROR_GROUP_INTERNAL,	6}	/* DRP_NOAVAIL */,
 	{ 0x1180044000020ull,	"Error: CVMX_PKI_GEN_INT[%llx][%u]",	0x44007,	0,	CVMX_ERROR_GROUP_INTERNAL,	7}	/* X2P_REQ_OFL */,
 	{ 0x1180044000040ull,	"Error: CVMX_PKI_ECC_INT0[%llx][%u]",	0x44040,	0,	CVMX_ERROR_GROUP_INTERNAL,	0}	/* ASM_SBE */,
 	{ 0x1180044000040ull,	"Error: CVMX_PKI_ECC_INT0[%llx][%u]",	0x44041,	0,	CVMX_ERROR_GROUP_INTERNAL,	1}	/* ASM_DBE */,
-- 
1.8.2.1

